hack/python-sdk/gen-sdk.sh

```diff
@@ -24,7 +24,7 @@ SWAGGER_JAR_URL="https://repo1.maven.org/maven2/org/openapitools/openapi-generat
 SWAGGER_CODEGEN_JAR="${repo_root}/hack/python-sdk/openapi-generator-cli.jar"
 SWAGGER_CODEGEN_CONF="${repo_root}/hack/python-sdk/swagger_config.json"
 SDK_OUTPUT_PATH="${repo_root}/sdk/python"
-VERSION=1.5.0
+VERSION=1.7.0rc0
```

Comment:
Uhm, the master branch is still 1.5.0 :(
@johnugeorge Can you submit a PR to the master branch to update the SDK version to v1.6.0?